client/data/mandatoryCourseWork.js

```diff
@@ -0,0 +1,6 @@
+["GitHomeworkFixErrors", "HTML-CSS-Coursework-Week1","HTML-CSS-Coursework-Week3", "HTML-CSS-Coursework-Week4",
```

Comment:
please put a name for it and also export it , so we can import it, like 
export const mandatoryCourswork =

Reply:
done